__tests__/test-GitIndex.js

```diff
@@ -117,4 +117,15 @@ describe('GitIndex', () => {
       expect(index.entriesMap.get('c').stages.length).toBe(1)
     })
   })
+
+  it('ensure nanoseconds are always present', async () => {
```

Comment:
> I think it would be a good idea to write a test that addresses this issue. We can create a repository with git that contains a file and add it to the test repos and then compare the isomorphic-git lstat values with the values in the index file.

@dead-end We can try to enhance this one but I don't know if we can hardcode timestamps to test them the way you want.

Reply:
If you create a repo with git, then this should have an index file with the timestamps. Call `GitIndex.fromBuffer` to get the index content and then call lstat for each entry and compare the values. I think this should be sufficient.

Reply:
@dead-end tried it with this approach: https://github.com/isomorphic-git/isomorphic-git/commit/e6c277ace38d134368566a4cf7c11952380dd94f but as 'nano.txt' is checked out on the build server, the timestamp changes, so the test will always fail.

Reply:
@jayree Sorry, I did not think of that. The only possibilities I can think of are calling git from node or put the nano.txt in a zipfile and unzip the file during the test run (this should preserve the timestamps). I do not know if this is possible.
@jcubic